.circleci/config.yml

```diff
@@ -133,7 +133,7 @@ jobs:
             - go-build-{{ arch }}-{{ checksum "go.sum" }}
       - run:
           name: Installing ko
-          command: go install github.com/google/ko@v0.11.1
+          command: go install github.com/google/ko@v0.11.2
```

Comment:
I also bumped the `ko` version because `v0.11.2` fixes a small bug that's specific to Go 1.18.